src/core/worker.js

```diff
@@ -197,6 +197,115 @@ IPDFStreamRangeReader.prototype = {
 }
 
 /** @implements {IPDFStream} */
+class PDFNetworkStream {
```

Comment:
Hmm..., I see. Okay, I will change it to the way we implement `class` in PDF.js, like: [IPDFStream](https://github.com/mozilla/pdf.js/blob/master/src/core/worker.js#L62). Although, I still think `class` looks nice :-)